superset-frontend/src/dashboard/components/nativeFilters/FilterBar/ActionButtons/index.tsx

```diff
@@ -53,7 +53,10 @@ const ActionButtonsContainer = styled.div`
     padding: ${theme.gridUnit * 4}px;
     padding-top: ${theme.gridUnit * 6}px;
 
-    background: linear-gradient(transparent, white 25%);
+    background: linear-gradient(
+      ${theme.transparent},
+      ${theme.colors.grayscale.light5} ${theme.opacity.mediumLight}
```

Comment:
```suggestion
      ${theme.colors.grayscale.light5} 0,
      ${theme.colors.grayscale.light5} ${theme.opacity.mediumLight}
```
Would this work just as well? Then we can use any color as our transparent color, rather than just white, right?

Reply:
Thanks! I tried it but unfortunately it does not work 😂